nio/responses.py

```diff
@@ -831,6 +838,20 @@ def from_dict(
         return cls(parsed_dict, event_type, state_key, room_id)
 
 
+class RoomGetEventResponse(ErrorResponse):
+    """A response indicating successful room get event request.
+
+    Attributes:
+        event (Dict): The requested event.
+    """
+
+    event: Dict = field()
```

Comment:
Shouldn't this be a `Event`? Or some subclass of it?

Reply:
Yeah fixed..